src/cml.js

```diff
@@ -235,17 +235,14 @@ class CML {
       };
 
       if (this.driver === GITHUB) {
+        const id = 'gh';
         if (data.includes('Running job')) {
-          const { id } = await getDriver(this).job({ time: date.getTime() });
           log.job = id;
           log.status = 'job_started';
-        } else if (
-          data.includes('Job') &&
-          data.includes('completed with result')
-        ) {
-          log.job = '';
+        } else if (data.includes('completed with result')) {
+          log.job = id;
           log.status = 'job_ended';
-          log.success = data.includes('Succeeded');
+          log.success = data.includes('Succeded');
```

Comment:
Is this a typo?

Reply:
And this? 😁 
```json
"status": {
                "running": 0,
                "succeeded": 1
              },
```

Reply:
According to [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/succeed?q=succeeded) and [Kubernetes](https://kubernetes.io/docs/concepts/workloads/controllers/job), `succeeded` is not a typo, if that's what you mean. 🙃

Reply:
succeded is not a typo. GH uses it

Reply:
https://en.wiktionary.org/wiki/succeded

Reply:
### [Wiktionary → succeded](https://en.wiktionary.org/wiki/succeded)
> Verb — **succeded**
> _simple past tense and past participle of [succede](https://en.wiktionary.org/wiki/succede)_
### [Wiktionary → succede](https://en.wiktionary.org/wiki/succede)
> Verb — **succede** (third-person singular simple present [succedes](https://en.wiktionary.org/wiki/succedes#English), present participle [succeding](https://en.wiktionary.org/wiki/succeding#English), simple past and past participle [succeded](https://en.wiktionary.org/wiki/succeded#English))
> 👉🏼 Dated👈🏼  form of [succeed](https://en.wiktionary.org/wiki/succeed#English)

Reply:
It is Succeeded indeed in GH... 🤦

Reply:
> _GH uses it_

```console
$ git clone --branch v2.288.1 https://github.com/actions/runner
$ grep --recursive --ignore-case succeded runner || echo not true
$ grep --recursive --ignore-case succeeded runner && echo not true
```

```console
2022-03-07 14:42:56Z: Listening for Jobs
2022-03-07 14:43:52Z: Running job: test
2022-03-07 14:44:00Z: Job test completed with result: Succeeded
```

https://github.com/actions/runner/blob/ad0d0c4d0a9e58943a688f66586b4dadf524ad9f/src/Runner.Listener/JobDispatcher.cs#L534
term.WriteLine($"{DateTime.UtcNow:u}: Job {message.JobDisplayName} completed with result: {result}");

https://github.com/actions/runner/blob/ad0d0c4d0a9e58943a688f66586b4dadf524ad9f/src/Runner.Common/Util/TaskResultUtil.cs#L21-L32
        public static TaskResult TranslateFromReturnCode(int returnCode)
        {
            int resultInt = returnCode - _returnCodeOffset;
            if (Enum.IsDefined(typeof(TaskResult), resultInt))
            {
                return (TaskResult)resultInt;
            }
            else
            {
                return TaskResult.Failed;
            }
        }


https://github.com/actions/runner/blob/be9632302ceef50bfb36ea998cea9c94c75e5d4d/src/Sdk/DTWebApi/WebApi/TaskResult.cs#L6-L25

Reply:
> _It is Succeeded indeed in GH..._

Whoops! Sorry, I missed your comment. Yes, it's `succeeded` and we should probably update our code accordingly.

Reply:
if it helps, I hate that GH uses "Canceled" :D